runtime/etc/logrotate.d/etcd

```diff
@@ -0,0 +1,11 @@
+/var/log/etcd.log
+{
+        rotate 1
```

Comment:
@szuecs What is the problem with this configuration? What would be your suggestion?
I always need the latest log on the machine, for older entries I can take a look also in other places.